packages/apollo-graphql/src/operationId.ts

```diff
@@ -69,25 +69,21 @@ export function defaultEngineReportingSignature(
 }
 
 // The operation registry signature function consists of removing extra whitespace,
-// sorting the AST in a deterministic manner, hiding string and numeric literals,
-// and removing unused definitions. This is a less aggressive transform than its
-// engine reporting signature counterpart.
-//
-// XXX
-// The hiding of literals is currently being discussed. This behavior
-// should not be depended on in the future, as it's likely we will choose not
-// to hide them at all. The rationale being, if queries are being shipped to
-// a client bundle, exposing PII via this signature is a very small concern,
-// relatively speaking.
+// sorting the AST in a deterministic manner, potentially hiding string and numeric
+// literals, and removing unused definitions. This is a less aggressive transform
+// than its engine reporting signature counterpart.
 export function defaultOperationRegistrySignature(
   ast: DocumentNode,
-  operationName: string
+  operationName: string,
+  options: { preserveStringAndNumericLiterals: boolean } = {
+    preserveStringAndNumericLiterals: false
+  }
```

Comment:
Introducing options to a method called `defaultOperationRegistrySignature` seems a bit off since a "default" should be sans options.  Could we rename this method to `operationRegistrySignature` and introduce another method using the existing name that merely calls this with an explicit `false` for `preserveStringAndNumericLiterals` in order to make the "default" intentions clear?

Reply:
Thanks! I made that change, though I'll note this may not actually be merge-able in its current state. The main issue is that the tests generally prefer the default value for `options`, but in some cases want to override it.

To avoid the duplication between tests and `defaultOperationRegistrySignature`, I left the default value in `operationRegistrySignature`, but that sort of defeats the purpose of `defaultOperationRegistrySignature`. My personal recommendation would be to deprecate `defaultOperationRegistrySignature` (I think that's part of the external API of this library?), but I'm curious what you think!

Tangentially, I noticed you're in Finland, which may make this back-and-forth take longer than it otherwise ought to. If you think it'd be easier, I'm totally open to you/the Apollo team taking over the PR from here, but I'm also happy to do another round of changes. As I've mentioned, my main goal is to get this shipped quickly to unblock my team. :)

Reply:
I believe you're right that `defaultOperationRegistrySignature` is a part of the API.  This LGTM, thank y'all.